src/components/WebCLI/Output/Output.tsx

```diff
@@ -130,15 +130,32 @@ const WebCLIOutput = ({
     // are then open it back up.
     if (
       // Only trigger this condition if the content changes.
-      content != contentPrevious &&
-      content.length > 1 &&
+      !fastDeepEqual(content, contentPrevious) &&
+      content.length &&
       height <= HELP_HEIGHT &&
       height !== DEFAULT_HEIGHT
     ) {
       setHeight(DEFAULT_HEIGHT);
     }
   }, [content, contentPrevious, height]);
 
+  let output: ReactNode = null;
+  if (command) {
```

Comment:
This PR was too big, so I split out the handlers for inserting links into the status tables which I'll propose once this PR has landed.